src/systems/ec/implementations/vanilla/packages/ec-component-file/ec-component-file.scss

```diff
@@ -19,10 +19,11 @@
 ) {
   .ecl-file {
     margin: 0;
+    max-width: 823px;
```

Comment:
Where is this coming from? (can't find it in the specs)
better way to define this?

Reply:
this value has been provided by the designers, in private message. I asked to put it in the specstoo, but it hasn't been done yet.
![image](https://user-images.githubusercontent.com/25579221/68443597-b1e06d00-0217-11ea-8337-d4109dcb6657.png)